web/locales/cs.yml

```diff
@@ -0,0 +1,6 @@
+cs:
+  'Recurring Jobs': Pravidelně opakované
```

Comment:
Could you please use key like strings for this. 
Example:
  recurring_jobs instead of 'Recurring Jobs'.

And add the en.yml. Thanks